meetup/models.py

```diff
@@ -0,0 +1,40 @@
+from django.db import models
+from pizzaplace.models import PizzaPlace
+from django.core.validators import RegexValidator
+from meetup.services.meetup_api_lookup_agent import MeetupApiLookupAgent
+from django.core.exceptions import ValidationError
+
+
+def validate_urlname(link):
+  validator = RegexValidator(
+    regex='meetup\.com\/\w+(-\w+)*\/$',
+    message="Does not conform to Meetup Url",
```

Comment:
Is this message as useful as it could be? Does it give the user all the information they need to understand and fix whatever's wrong with the meetup URL?